TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/VNext/Requests/SetProfessionalStatusRequest.cs

```diff
@@ -6,17 +6,17 @@ namespace TeachingRecordSystem.Api.V3.VNext.Requests;
 
 public record SetProfessionalStatusRequest
 {
-    public required Guid RouteTypeId { get; init; }
-    public required ProfessionalStatusStatus Status { get; init; }
-    public DateOnly? AwardedDate { get; init; }
-    public DateOnly? TrainingStartDate { get; init; }
-    public DateOnly? TrainingEndDate { get; init; }
-    public Option<string[]> TrainingSubjectReferences { get; init; }
-    public SetProfessionalStatusRequestTrainingAgeSpecialism? TrainingAgeSpecialism { get; init; }
-    public string? TrainingCountryReference { get; init; }
-    public string? TrainingProviderUkprn { get; init; }
-    public Guid? DegreeTypeId { get; init; }
-    public Guid? InductionExemptionReasonId { get; init; }
+    public required Guid RouteTypeId { get; set; }
```

Comment:
Why are these no longer `init`s?

Reply:
to be able to configure in tests i.e. set individual property values

Reply:
Can you use `with` e.g. `var request = originalRequest with { OverridenProperty = value }`;?